logpyle/__init__.py

```diff
@@ -251,11 +262,18 @@ def __init__(self, dt: Optional[float], name: str, unit: str = None,
 
 
 class PushLogQuantity(LogQuantity):
+    """A loggable scalar whose value can be updated.
```

Comment:
This should comment a bit on how the `LogQuantity` interface is a "pull"  model generically, and how this offers "push" capability over it. What happens to redundant "set" attempts? What if nothing is pushed in a cycle?